src/content/guides/prepare-azure-account-for-guest-clusters/index.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Prepare an Azure account to run Giant Swarm guest clusters"
+description = "This guide will walk you through all necessary steps to set up an Azure account with approriate role definition and service principal for operating Giant Swarm guest clusters."
+date = "2018-07-18"
+type = "page"
+weight = 100
+tags = ["tutorial"]
++++
+
+# Prepare an Azure account to run Giant Swarm guest clusters
+
+In a Giant Swarm installation the guest clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription (or account) from the host cluster. This gives great flexibility depending on the requirements and the usage scenario. For example, it allows the host cluster to be running in an Azure subscription owned by Giant Swarm, while guest clusters operate in different Azure subscriptions each, depending on a customer’s department using them.
+Giant Swarm operate guest clusters using a service called `azure-operator` which runs on the host cluster.
+
+## Overview
+
+In order to run Giant Swarm guest clusters, an Azure subscription need the following elements :
+
+- Role definition: set of permission to operate guest clusters in the Azure subscription.
+- Service principal: an identity (bound to the previoulsy defined role definition) to access the Azure subscription.
+
+## Create Azure role definition & service principal
+
+In order to perform necessary actions to deploy and maintain guest clusters in your Azure subscription, `azure-operator` needs to access the subscription using a service principal.
+Here we descirbe all the steps to set it up.
+
+1. Prerequisites
+
+To create a service principal you need:
+
+- An account with [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [User Access Administrator](User Access Administrator) role
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed
+
+2. Role definition
+
+Download our [role definition](https://raw.githubusercontent.com/giantswarm/azure-operator/38caa99efac9db440433c73646de54a5478f8cb6/policies/guest.json).
+
+Open it and replace `${SUBSCRIPTION_ID}` with your subscription id, you can find it [here](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade) using the portal, or using Azure cli `az account list --output table`.
+
+Create the role definition : `az role definition create --role-definition @/path/to/guest.json`.
+
+3. Service principal
+
+Create the service principal : `az ad sp create-for-rbac --name "azure-operator-sp" --role "azure-operator" --query '{client_id:appId, secret_key:password, tenant_id:tenant, subscription_id:""}'`
+
```

Comment:
That section doesn't render well. See

![image](https://user-images.githubusercontent.com/273727/43518793-6aa72f98-958d-11e8-8f47-cf1f072baf89.png)

I recommend to prepend the step headlines with `###` to make them real headlines instead of list items.

```
### 1. Prerequisites

...

### 2. Role definition

...
```